### PR TITLE
add offsets calculation to MegatronGPTModel.complete method

### PR DESCRIPTION
**Pull request by dimapihtar**
Signed-off-by: dimapihtar <dpykhtar@nvidia.com>

### CODE REVIEW

**Review by okuchaiev — APPROVED** [review]
lgtm

File: examples/nlp/language_modeling/megatron_gpt_eval.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def main():
     parser = ArgumentParser()
     parser.add_argument("--model_file", type=str, default="", required=True, help="Pass path to model's .nemo file")
     parser.add_argument(
-        "--prompt", type=str, default="", required=True, help="Prompt for the model (a text to complete)"
+        "--path_to_file", type=str, default="", required=True, help="Path to file with prompts (a text to complete)"
```
Comment (ericharper) [inline examples/nlp/language_modeling/megatron_gpt_eval.py:36]: Do we have to remove `--prompt` ? It's nice to be able to feed in some text from the CLI.

**Review by ericharper — CHANGES_REQUESTED** [review]
There's a binary file that needs to be removed from the PR: examples/nlp/language_modeling/.megatron_gpt_eval.py.swp

File: examples/nlp/language_modeling/megatron_gpt_eval.py (hunk 1 above)
Comment (dimapihtar) [reply examples/nlp/language_modeling/megatron_gpt_eval.py:36]: Should I create a separate `.py` example for evaluation prompts from file?

Comment (okuchaiev) [reply examples/nlp/language_modeling/megatron_gpt_eval.py:36]: @dimapihtar  let's just have both "--prompt" and "--path_to_file" options and require at least one to be specified. If both are specified, use "--prompt" and ignore "--path_to_file"

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**